### ISSUE TITLE
Background color and merged columns

### ISSUE THREAD

**Issue by bonafernando** [eric-wood/excel2latex#23]
Can you consider background color and merged columns?

I was expecting this:
![image](https://user-images.githubusercontent.com/2565620/61421001-7eca4480-a8db-11e9-9b41-6fdb0c213594.png)

And got this:
![image](https://user-images.githubusercontent.com/2565620/61421012-85f15280-a8db-11e9-9f09-02d41192abed.png)

**Comment by eric-wood** [comment]
possibly! I had a rewrite going at one point but I've been really busy the last few years and haven't had much time to devote to it; the background colors I _think_ wouldn't be horrible to implement, but merged columns might be a bit of work.

I rolled my own XLSX parser, and it turns out the file format is extremely complicated. So far I haven't seen any off-the-shelf solutions that provide this level of info for parsing. The largest issue people complain about is the fact that this tool doesn't understand number formatting, but I still haven't even seen a library that does :(

Maybe one day I will close this issue, but it might be a while.

**Comment by bonafernando** [comment]
Thanks for the quick response @eric-wood. No worries, it was a small table and I managed to make it manually hehehe. I opened this issue mostly to contribute for future features and/or users of your software, so it's not a priority and I'm not blocked by it. hehehe
But don't stop. Keep doing it. I can easily imagine google docs using .tex as a Google Sheets downloadable option someday. And your work is really relevant for that matter.
